arithmetization/src/main/java/net/consensys/linea/zktracer/module/wcp/Wcp.java

```diff
@@ -34,12 +34,20 @@
 import org.apache.tuweni.bytes.Bytes;
 import org.apache.tuweni.bytes.Bytes32;
 import org.hyperledger.besu.evm.frame.MessageFrame;
+import org.hyperledger.besu.evm.worldstate.WorldView;
 
-@RequiredArgsConstructor
 public class Wcp implements Module {
   private final StackedSet<WcpOperation> operations = new StackedSet<>();
+  /** count the number of rows that could be added after the sequencer counts the number of line */
+  public final Stack<Integer> additionalRows = new Stack<>();
```

Comment:
Consider using `Deque<Integer> stack = new ArrayDeque<>();`, Stack extends Vector, thus not efficient at all.